### PR TITLE
Update copyright headers

### PR DESCRIPTION
**Pull request by ptomsich**
The LICENCE had gotten out-of-sync (copyright year, missing myself and VRULL as contributors) and some files missed the copyright headers.  This updates the LICENCE file and reruns apply_headers.

### CODE REVIEW

File: LICENCE [hunk 1]
```diff
@@ -22,6 +22,8 @@ Copyright (c) 2017-2021
   Microsoft, for contributions by Robert Norton-Wright and Nathaniel Wesley Filardo
   Peter Rugg
   Aril Computer Corp., for contributions by Scott Johnson
+  Philipp Tomsich
+  VRULL GmbH, for contributions by its employees
```
Comment (jrtc27) [inline LICENCE:26]: Are you covered by the latter or not? Also don't know if "its employees" is sufficient here, so far everyone has been explicitly named

Comment (ptomsich) [reply LICENCE:26]: Technically not, as control contravenes employment in our geography.
We could make it "employees and contractors", though...

**Comment by ptomsich** [comment]
Rebased onto latest  master (incl. the squashed Zfa).

@jrtc27 OK to merge with the three individual commits (i.e., w/o squashing the first two into a single update of LICENCE)?

**Comment by jrtc27** [comment]
That breakdown makes sense to me, so if the copyright is correct then yes

**Pull request closed without merging**